indexing-service/src/main/java/org/apache/druid/indexing/overlord/ImmutableWorkerInfo.java

```diff
@@ -225,6 +253,89 @@ private int getWorkerParallelIndexCapacity(double parallelIndexTaskSlotRatio)
     return workerParallelIndexCapacity;
   }
 
+  public boolean canRunTask(Task task, Map<String, Number> taskLimits)
```

Comment:
Can you please add java docs for this method.

Reply:
Done